main.go

```diff
@@ -95,30 +111,72 @@ func main() {
 
 	ctrl.SetLogger(zap.Logger(enableDebugLog))
 
-	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:             scheme,
-		MetricsBindAddress: metricsAddr,
-		LeaderElection:     enableLeaderElection,
-	})
-	if err != nil {
-		setupLog.Error(err, "unable to start manager")
-		os.Exit(1)
-	}
-
 	nsSlice := func(ns string) []string {
 		trimmed := strings.Trim(strings.TrimSpace(ns), "\"")
 		return strings.Split(trimmed, ",")
 	}
 
-	watchNs := make(map[string]bool)
-	if len(watchNamespaces) > 0 {
-		for _, ns := range nsSlice(watchNamespaces) {
-			watchNs[ns] = true
+	// If watchNamespaces is set ignore excludeNamespaces
+	if len(excludeNamespaces) > 0 && len(watchNamespaces) <= 0 {
```

Comment:
mmm i found this kind of misleading,
if **watchNamespaces** is set we'll go to the **else** code block where **excludeNamespaces** is **not** ignored, right?, 

i understand this as follows... "If we just have **excludeNamespaces**...." maybe that is more expressive, what do you think?